index.js

```diff
@@ -47,7 +47,8 @@ module.exports = {
         projectName: this.parent.pkg.name,
         projectTag: this.parent.pkg.version,
         projectHref: info && info.browse(),
-        deployVersion: 'ADDON_DOCS_DEPLOY_VERSION'
+        deployVersion: 'ADDON_DOCS_DEPLOY_VERSION',
+        searchTokenSeparator: /\s+/
```

Comment:
Unless I'm mistaken, a regex won't survive the `JSON.stringify` pass that happens when config is serialized for the app. Can we add a test to verify this is working?

Reply:
Works fine for the dummy app:

<img width="382" alt="screen shot 2018-06-11 at 15 35 04" src="https://user-images.githubusercontent.com/11009/41238082-0fcc38a2-6d8d-11e8-876c-bd715d6f98b6.png">

I'll get a test in place.

Reply:
Added. I'm open to a different location for the test.

Reply:
It seems like that's working because the search terms aren't being split at all — that means if I search e.g. `render multiple` with your changes, I get no results, but on the latest deployed version of the site, that returns several options back.

It looks like the regex is being encoded as `{}` in the serialized config:

![screen shot 2018-06-11 at 1 00 46 pm](https://user-images.githubusercontent.com/108688/41245940-7df756a6-6d77-11e8-8794-71a3445d15ba.png)